docs/wmco-prerequisites.md

```diff
@@ -0,0 +1,45 @@
+# Cluster and OS Pre-Requisites
+Below are the Cluster and OS prerequisites for WMCO. Please also see the vSphere-specific section that 
+may be relevant.
+
+## Supported Kubernetes Distribution Version
+* OKD/OCP 4.6+ must be used in order to configure Windows VMs
+* OKD/OCP 4.8+ and WMCO 3.0+ must be used in order to configure Bring-Your-Own-Host(BYOH) Windows instances
+
+## Supported Cloud Providers based on OKD/OCP Version and WMCO version
+| Cloud Provider           | Supported OKD/OCP Version   | Supported WMCO version   |
+| -----------              | -----------                 | -----------              |
+| Amazon Web Services (AWS)| 4.6+                        | WMCO 1.0+                |
+| Azure                    | 4.6+                        | WMCO 1.0+                |
+| VMware vSphere           | 4.7+                        | WMCO 2.0+                |
+
+## Supported Windows Server versions
+The following table outlines the supported
+[Windows Server version](https://docs.microsoft.com/en-us/windows/release-health/release-information) based on the 
+applicable cloud provider.
+
+Note: Any unlisted Windows Server version are NOT supported, and will cause errors. To prevent 
+these errors, only use the appropriate version according to the cloud provider in use. 
+
+| Cloud Provider      | Supported Windows Server version                                        |
+| -----------         | -----------                                                             |
+| AWS                 | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809  |
+| Azure               | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809  |
+| VMware vSphere      | Windows Server Semi-Annual Channel (SAC): Windows Server 2004           |
+
+## Supported Networking
```

Comment:
Please add the custom VXLAN port requirement for vSphere. In addition a table to show which Windows version works with the custom VXLAN port.

Reply:
Will do. Which custom port can they can use? [OpenShift docs](https://docs.openshift.com/container-platform/4.7/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html#configuring-hybrid-ovnkubernetes) say "The custom port can be any open port excluding the default 4789 port" but our [dev doc](https://docs.google.com/document/d/1rU7VRP47JPPmPNeKmUMmInJepcH20pk2J2Xz1EEy4iQ/edit#) says "Custom port can range from 9000-9999"

Reply:
I am not so sure if any port other than 4789 can be used in all providers. 9000-9999 is the recommendation from the SDN team based on what ports were open for sure on AWS. Just link to the GitHub doc for now: https://github.com/openshift/windows-machine-config-operator/blob/master/docs/setup-hybrid-OVNKubernetes-cluster.md#vsphere